source/components/other-helper-classes.md

```diff
@@ -1,10 +1,10 @@
-title: Other CSS Helper Classes
+Other CSS Helper Classes
```

Comment:
Every other guide / component page has the `title: ` prefix, so I'd think that this page should not break from the norm.